files/fr/glossary/code_splitting/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Code splitting
```

Comment:
General note: code splitting = division de code ?
You can still add `(en anglais <i lang="en">code splitting</i>)` for the first occurence in the introduction paragraph :)

Reply:
I firstly translate i into "découpage de code" but "division de code" is more accurate indeed. I decided to let the english version like in this article : https://blog.eleven-labs.com/fr/react-d%C3%A9buter-le-code-splitting. So i should i translate every occurency or just the first one? 
Adding ```(en français <i lang="fr">division de code</i>)``` for the moment, waiting for your feedback, thank you!

Reply:
This `<p><strong>Le code splitting</strong> (en français <i lang="fr">division de code</i>)`
doesn't work.
The lang attribute is used to indicate when the text suddenly use a different language than the language of the current page. So you don't have to use `lang="fr"` on a page written in French (which is the case).
Please replace it with something like this:
`<p><strong>Le <i lang="en">code splitting</i> (en français «&nbsp;division de code&nbsp;»)`